src/js/models/Trivia.js

```diff
@@ -0,0 +1,24 @@
+import {createQuestion} from "../view/createQuestion"
+export const triviaStart = document.querySelector('.pokemonTrivia__container');
+export class Trivia {
+    constructor(parent){
+        const pokemonTrivia__title = document.createElement('h1')
+        pokemonTrivia__title.classList.add("pokemonTrivia__title")
```

Comment:
in javascript we always use conventions in this case is camelCase
words will be like pokemonTriviaTitle never with dashes or undercores or anything else it is a convention that we must follow always in code

Reply:
done